alma/alma-8.x/alma-8.6-hpc/install.sh

```diff
@@ -52,6 +52,9 @@ $COMMON_DIR/install_hpcdiag.sh
 #install monitoring tools
 $COMMON_DIR/../alma/common/install_monitoring_tools.sh
 
+# install Azure/NHC Health Checks
+$UBUNTU_COMMON_DIR/install_health_checks.sh
```

Comment:
Can you please also add this to the 8.7 tree?

Reply:
addressed